src/sentry/hybridcloud/models/webhookpayload.py

```diff
@@ -93,7 +114,8 @@ def get_attributes_from_request(
     def create_from_request(
         cls,
         *,
-        region: str,
+        destination_type: DestinationType,
+        region: str | None,
```

Comment:
Probably we can exclude these changes. I think that the non-optional parameter will break existing calls